app/graphql/mutations/user_books/patch_user_book.rb

```diff
@@ -0,0 +1,29 @@
+module Mutations
+  module UserBooks
+    class PatchUserBook < Mutations::BaseMutation 
+      argument :user_id, Integer, required: true
+      argument :book_id, Integer, required: true 
+      argument :borrower_id, Integer, required: true
+      argument :status, Integer, required: true
+
+      field :success, String, null: false
+      
+      def resolve(user_id:, book_id:, borrower_id:, status:)
+        user_book = UserBook.find_by(user_id: user_id, book_id: book_id)
+
+        if user_book.nil?
+          {
+            success: 'False'
+          }
+        else
+          user_book.status = status
+          user_book.borrower_id = borrower_id
+          user_book.save
+          {
+            success: 'True'
+          }
+        end
```

Comment:
And then same here, I know front end probably doesn't need anything but thinking about posterity, maybe we should add a user_book field (Types::UserBookType) just in case they want more info / confirmation of what was changed. Plus we have that type all set up already anyway.